stable/artifactory-ha/templates/artifactory-binarystore.yaml

```diff
@@ -0,0 +1,149 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: artifactory-binarystore
```

Comment:
Use the fullname template for resource names. Apply everywhere.

Reply:
Please do it for all templates. In general, it should be possible to install charts multiple times. Names could collide if you don't use the fullname template.